scripts/__tests__/changelog-generator.test.js

```diff
@@ -165,3 +169,60 @@ describe("functions that hit GitHub's commits API", () => {
     });
   });
 });
+
+/**
+ * @param {string} sha
+ */
+function getCommitMessage(sha) {
+  return git(RN_REPO, "log", "--format=%B", "-n", "1", sha);
+}
+
+/**
+ * @typedef {object} PartialPlatformChanges
+ * @property {string[]=} android
+ * @property {string[]=} ios
+ * @property {string[]=} general
+ */
+
+/**
+ * @typedef {object} PartialChanges
+ * @property {PartialPlatformChanges=} breaking
+ * @property {PartialPlatformChanges=} added
+ * @property {PartialPlatformChanges=} changed
+ * @property {PartialPlatformChanges=} deprecated
+ * @property {PartialPlatformChanges=} removed
+ * @property {PartialPlatformChanges=} fixed
+ * @property {PartialPlatformChanges=} security
+ * @property {PartialPlatformChanges=} unknown
+ */
+
+describe("formatting and attribution regression tests", () => {
```

Comment:
Should be straightforward to add entries you find while reviewing the CHANGELOG.